dialogue/local-abundance.md

```diff
@@ -0,0 +1,45 @@
+
+S: What did you do?
+
+P: I became like water.
+
+S: How did you become like water?
+
+P: You are a very curious person, I’m starting to like you. Growing up with the Internet, I started to understand I could reverse engineer my life and become whatever I wanted to. Once I realized that there’s not really any limitation, I decided to become like water.
+
+S: That’s a peculiar choice.
+
+P: I’ve always looked at life as a choice between intellectual honesty and ego. Intellectual honesty is just a fancy way to say “play”. Ego is everything we do to seek status. Just by observing adults, I realized many of them are just seeking status. Honestly, they look miserable; I’m quite good at spotting sad people. Kids, on the other hand, have this playful view of the world. It’s something very intrinsic and doesn't come from ego. Ego is the worst thing in life, it destroys everything. People usually develop an ego when they start compromising on things they love in order to do things they don’t want to do. Eventually, this “ego” thing starts growing, and people start pretending to love what they do. That’s why I became water. Water has no ego and seeks no [status](https://twitter.com/NavalismHQ/status/1557575533730430976). Water flows.
+
+S: I agree with you, ego is the worst thing we can have. However, you mentioned earlier that people assume your life is like a blossoming garden due to your lusory attitude. Isn’t a blossoming garden more exciting than water?
+
+P: I see my life more like dirt, as opposed to a blossoming garden. Dirt gives you the opportunity to grow new things, it makes room for surprises. I’m more interested in the process than in the outcome. That’s why my name is Play.
+
+S: But still, I have one question left. Since you became like water, have you found seeds to nurture?
+
+P: I’m starting to find them. They were always here, but somehow I was distracted. They say “think globally, act locally”, right? I always believed that but, due to the Internet, we can act globally in a permissionless way. For quite some time, I was looking for places of local abundance. It’s like I was looking for a Schelling Point for all playful actors. We are all unpaid actors in this game of life. Recently, I found out that the Internet is a beautiful Schelling Point for us. It made local abundance a global place. We’re all together in this campfire now! We can play, we can build, we can create, we can discuss ideas. 
```

Comment:
Would love a link for Schelling Point

Reply:
link added! (from wikipedia, it was the most credible-neutral source)